### PR TITLE
Internal users confirm modal

### PR DESCRIPTION
**Pull request by sergibondarenko**
This closes
https://floragunn.atlassian.net/browse/ITT-2100

### CODE REVIEW

**Comment by sergibondarenko** [comment]
Screenshots
Screenshot 2019-05-06 at 17 27 04
Screenshot 2019-05-06 at 17 27 48

File: public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js [hunk 1]
```diff
@@ -168,7 +178,7 @@ class InternalUsers extends Component {
     }
 
     const onClick = () => {
```
Comment (mgustafsson1) [inline public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js]: Can we rename this to something that describes what the onClick does?

File: public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js [hunk 2]
```diff
@@ -168,7 +178,7 @@ class InternalUsers extends Component {
...
-      this.deleteUsers([...tableSelection.map(item => item.id)]);
+      this.handleUsersDelete([...tableSelection.map(item => item.id)]);
```
Comment (mgustafsson1) [inline public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js]: Is the ...-operator needed here?

File: public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js [hunk 3]
```diff
@@ -198,9 +208,26 @@ class InternalUsers extends Component {
...
+      <EuiConfirmModal
+        title={i18nConfirmDeleteText}
+        onCancel={() => this.handleUsersDelete([])}
+        onConfirm={() => this.deleteUsers(userIdsToDelete)}
+        cancelButtonText={i18nCancelText}
+        confirmButtonText={i18nConfirmText}
+        buttonColor="danger"
+        defaultFocusedButton={EUI_MODAL_CONFIRM_BUTTON}
+      >
+        <p>{i18nDoYouReallyWantToDeleteText} {userIdsToDelete.join(', ')}?</p>
```
Comment (mgustafsson1) [inline public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js]: Is there any support for pluralization in the i18n-library?
Like "Do you really want to delete user1, user2 and user3?"

File: public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js (hunk 2 above)
Comment (sergibondarenko) [reply public/apps/configuration-react/pages/InternalUsers/containers/InternalUsers/InternalUsers.js]: You are right, the destructuring here is excessive, it is enough to do
```
this.handleUsersDelete(tableSelection.map(item => item.id));
```

**Comment by sergibondarenko** [comment]
@mgustafsson1 do you approve this PR now?

**Comment by mgustafsson1** [comment]
> @mgustafsson1 do you approve this PR now?

@sergibondarenko Approved.

**Pull request merged**